locale/inc/leatherman/locale/locale.hpp

```diff
@@ -15,12 +15,11 @@
 #include <locale>
 #include <vector>
 #include <functional>
-
-#ifdef LEATHERMAN_I18N
```

Comment:
Yes, I realized this and will fix. My initial motivation was to have less conditionally compiled code in this file--magical stuff that happens when LEATHERMAN_I18N is on or off--but obviously this needs to work correctly without boost::locale.